pkg/datapath/linux/config/config.go

```diff
@@ -109,8 +113,10 @@ func (h *HeaderfileWriter) WriteNodeConfig(w io.Writer, cfg *datapath.LocalNodeC
 		fw.WriteString(FmtDefineAddress("NAT46_PREFIX", nat46Range.IP))
 	}
 
-	extraMacrosMap["HOST_IP"] = hostIP.String()
-	fw.WriteString(defineIPv6("HOST_IP", hostIP))
+	if option.Config.EnableIPv6 {
+		extraMacrosMap["HOST_IP"] = hostIP.String()
```

Comment:
I don't think this line is suppose to be conditional by the `option.Config.EnableIPv6`

Reply:
It's only used in `ENABLE_IPV6` paths:

```
$ git grep HOST_IP bpf/
bpf/lib/encap.h:                BPF_V6(host_ip, HOST_IP);
bpf/lib/encap.h:                BPF_V6(host_ip, HOST_IP);
bpf/node_config.h:DEFINE_IPV6(HOST_IP, 0xbe, 0xef, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0x0, 0x0, 0xa, 0x0, 0x2, 0xf, 0xff, 0xff);
```